docs/user/how-to/write-plans.rst

```diff
@@ -0,0 +1,203 @@
+Writing bluesky plans for blueapi
+=================================
+
+**Please read the following completely and carefully, as you risk losing data if plans are written incorrectly.**.
+
+For an introduction to bluesky plans and general forms/advice, `see the
+bluesky documentation <https://nsls-ii.github.io/bluesky/plans.html>`__.
+Blueapi has some additional requirements, which are explained below.
+
+Generally, if a ``MsgGenerator`` includes at least one
+``open_run`` and ``close_run``, it is a ``plan``, a complete description
+of an experiment. If it does not, it is a ``stub``.
+
+Plans
+~~~~~
+
+A ``plan`` is a complete data collection proceedure.
+
+Type Annotations
+^^^^^^^^^^^^^^^^
+
+“Plans” in the bluesky context are any iterable of ``Msg`` instructions,
+but in the context of blueapi are ``PlanGenerator``\ s: functions that
+take any arguments and return a ``MsgGenerator`` (a python ``Generator``
+that yields ``Msg``\ s). **These ``PlanGenerator``\ s must be annotated
+with the return type ``MsgGenerator`` to be added to the blueapi
+context**.
```

Comment:
The ``Words`` aren't rendering properly for some reason? Maybe they don't in bold?
See https://diamondlightsource.github.io/blueapi/writing-plans-readme/user/how-to/write-plans.html

Reply:
https://docutils.sourceforge.io/docs/dev/rst/alternatives.html#nested-inline-markup

Looks like nested markup is an outstanding todo since 2001. I'll see how it renders in a Warning box.